### PR TITLE
Organisation profile page redesign

### LINKED ISSUE #18645: Left/Right sidebars on org page can have similar UI as that of profile-page
## Current Situation

**Org Page**

[Reference](https://dev.to/devteam)
Org page

**Profile Page**

[Reference](https://dev.to/ben)
Profile Page

The difference between both the screens is that the left/right side bar content on profiles is wrapped in a card-container and on org page they are not.

**Suggestion**: We should wrap content on org pages on left/right sidebars in card-containers similar to that of profile page.

**Note**: This is just a suggestion and has not been finalised internally yet.

### PR DESCRIPTION
**Pull request by rt4914**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [ ] Feature
- [ ] Bug Fix
- [x] Optimization
- [ ] Documentation Update

## Description

This PR redesigns the organisation page to look more like similar to user-profile page

### How to test

Locally visit any profile like http://localhost:3000/org2198 or http://localhost:3000/org1056 or you can create your own organisation with full details by visiting http://localhost:3000/settings/organization

## Related Tickets & Documents

- Related Issue #18645 
- Closes #18645 

### Previous Design

Screenshot 2022-11-10 at 10 57 18 PM

### Main changes

1. The header container which show organisation details like logo, name, email, etc has been changed such that the width of item is similar to that of any user-profile page. To test this you can compare the `header` of any user-profile with org-profile
2. We have removed the right-side bar also known as sidebar-additional and shifted its content to left side bar
3. All items of left side bar have been redesigned to show like cards.

### Output

https://user-images.githubusercontent.com/9396084/201165936-f35bb164-8b67-46c8-8959-306bb85d584e.mov

Screenshot 2022-11-10 at 10 52 59 PM
Screenshot 2022-11-10 at 10 53 06 PM
Screenshot 2022-11-10 at 10 53 19 PM

### UI accessibility concerns?

None

## Added/updated tests?

- [x] Yes
- [ ] No, and this is why: _please replace this line with details on why tests
      have not been included_
- [ ] I need help with writing tests

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

- [ ] I've updated the [Developer Docs](https://developers.forem.com) or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
- [ ] This PR changes the Forem platform and our documentation needs to be
      updated. I have filled out the
      [Changes Requested](https://github.com/forem/admin-docs/issues/new?assignees=&labels=&template=changes_requested.md)
      issue template so Community Success can help update the Admin Docs
      appropriately.
- [ ] I've updated the README or added inline documentation
- [ ] I've added an entry to
      [`CHANGELOG.md`](https://github.com/forem/forem/tree/main/CHANGELOG.md)
- [ ] I will share this change in a [Changelog](https://forem.dev/t/changelog)
      or in a [forem.dev](http://forem.dev) post
- [ ] I will share this change internally with the appropriate teams
- [ ] I'm not sure how best to communicate this change and need help
- [x] This change does not need to be communicated, and this is why not: _please
      replace this line with details on why this change doesn't need to be
      shared_

## [optional] Are there any post deployment tasks we need to perform?

No

### CODE REVIEW

**Comment by rt4914** [comment]
@jaw6 @lboogie04 @Ridhwana This PR is ready for review.

**Review by Ridhwana — COMMENTED** [review]
@rt4914 I've tested this and it looks really good ✨. I left one question below.

File: app/assets/stylesheets/widgets.scss [hunk 1]
```diff
@@ -279,6 +279,36 @@
   }
 }
 
+.org-sidebar-widget {
```
[the remark this answers is not in the document]
Comment (Ridhwana) [reply app/assets/stylesheets/widgets.scss:282]: @rt4914 thanks so much for the in depth explanation. I think its okay to have a combination of crayons classes and a custom class (so in this case things like `block` remain as a crayons class in the hml), if a crayons class is unable to solely fulfill the requirements of the design.  I think I've seen this done in other places but let's park this discussion for now, I've added it as a point to chat about when we dive into the best practices endeavour that I'm working on.

**Review by Ridhwana — APPROVED** [review]
Thanks @rt4914 this looks great! ✨

**Review by jaw6 — APPROVED** [review]

**Pull request merged**